kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/embedder/MavenProjectLoader.java

```diff
@@ -97,15 +98,23 @@ public static MavenEmbedder newMavenEmbedder(boolean offline) {
         return mavenEmbedder;
     }
 
+    public static String getGlobalRepo(){
+        String global = System.getProperty(GLOBAL_M2_REPO_URL);
```

Comment:
There is a overloaded version of this method that you can use to avoid the if:
`System.getProperty("property.name", "default value when property.name is undefined")`

Reply:
Fixed